.github/workflows/labels-sync.yml

```diff
@@ -0,0 +1,17 @@
+name: Sync Labels
+on:
+  push:
+    paths:
+      - ".github/labels.yml"
+      - ".github/workflows/labels-sync.yml"
+    branches:
+      - master
+jobs:
+  build:
+    name: Sync labels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: micnncim/action-label-syncer@v1.3.0
```

Comment:
I'd pin this to a hash to make it more secure.

Reply:
Done!